docs/bodyswap.rst

```diff
@@ -14,15 +14,27 @@ Usage
 ::
 
     bodyswap [--unit <id>]
+    bodyswap linger
 
 If no specific unit id is specified, the target unit is the one selected in the
 user interface, such as by opening the unit's status screen or viewing its
-description.
+description. Otherwise, a valid list of units to bodyswap into will be shown.
+If bodyswapping into an entity that has no historical figure, a new historical figure is created for it.
+If said unit has no name, a new name is randomly generated for it, based on the unit's race.
+If no valid language is found for that race, it will use the DIVINE language.
+
+If you run bodyswap linger, the killer is identified by examining the historical event generated
```

Comment:
I think you need to mention that `bodyswap linger` must be run immediately after you have died, and that it will put you into the body of your killer.